packages/compile-solidity-tests/test/compilerSupplier/loadingStrategies/VersionRange.ts

```diff
@@ -58,32 +58,38 @@ const allVersions = {
   latestRelease: "0.5.4"
 };
 
+const unStub = (stubbedThing: object, methodName: string): void => {
+  stubbedThing[methodName].restore();
+};
+
 describe("VersionRange loading strategy", () => {
   beforeEach(function () {
-    sinon.stub(instance, "getSolcVersionsForSource").returns(allVersions);
+    sinon.stub(instance, "getSolcVersionsForSource").returns(Promise.resolve(allVersions));
   });
   afterEach(function () {
-    instance.getSolcVersionsForSource.restore();
+    unStub(instance, "getSolcVersionsForSource");
   });
 
   describe("async load(versionRange)", () => {
     beforeEach(() => {
       sinon.stub(instance, "getCachedSolcByVersionRange");
       sinon.stub(instance, "getSolcFromCacheOrUrl");
-      sinon.stub(instance, "versionIsCached").returns(true);
+      sinon.stub(instance, "versionIsCached").returns(undefined);
     });
     afterEach(() => {
-      instance.getCachedSolcByVersionRange.restore();
-      instance.getSolcFromCacheOrUrl.restore();
-      instance.versionIsCached.restore();
+      unStub(instance, "getCachedSolcByVersionRange");
+      unStub(instance, "getSolcFromCacheOrUrl");
+      unStub(instance, "versionIsCached");
     });
 
     it("calls getCachedSolcByVersionRange when single solc is specified", async () => {
       await instance.load("0.5.0");
+      // @ts-ignore - TS not smart enough to recognize stubbed methods
```

Comment:
OK, this one (and the one right after it) were pre-existing, but I'm guessing these should be using `assert.isDefined`?

Reply:
Oh is it?  I didn't get that.  For booleans there's also `isTrue` which is more specific. :)